File: CodeWars/JavaEx/Order.java
public class Order {

  public static String order(String words) {

    if (words.isEmpty()) {
      return "";
    }

    StringBuilder sb = new StringBuilder();

    for (int i = 1; i < 10; i++) {
      String[] array = words.split(" ");
      for (String arr : array) {
        if (arr.contains(String.valueOf(i))) {
          sb.append(arr + " ");
          break;
        }
      }
    }

    return sb.toString().trim();
  }

}

File: CodeWars/JavaScriptEx/Order.js
function order(words) {
  if (words.length === 0) {
    return "";
  }

  let result = "";

  for (let i = 1; i <= 9; i++) {
    const array = words.split(" ");
    for (let j = 0; j < array.length; j++) {
      if (array[j].includes(i)) {
        result += array[j] + " ";
        break;
      }
    }
  }

  return result.trim();
}
